src/index.js

```diff
@@ -1,10 +1,19 @@
 // import _ from './lodash';
+import './style.css';
+import rel from './pics/reload.png';
+import ent from './pics/enter.png';
 import loader from './Modules/loader.js';
 import { addtask } from './Modules/add.js';
 
 const message = document.querySelector('.message');
 message.style.display = 'none';
 
+const img = document.getElementById('reload');
+img.src = rel;
+
+const img2 = document.getElementById('enter');
+img2.src = ent;
+
 loader();
 
 const task = document.getElementById('inputTask');
```

Comment:
You seem to be moving ahead of schedule.  As shown in the project requirements below (point 3), you need to set an array of some simple to do tasks (array of objects) in this file, then write a function to iterate over the tasks array and populate an HTML list item element for each task. Remember that the index.html file should have an empty todo list placeholder (`<div>` or `<ul>` element ) so that on page load you render the dynamically created lists of tasks from the index.js file inside. :wink: 
 Kindly take your time, go through the project requirements to help you get a grasp of what you are expected to do in this milestone. 

![Screenshot from 2023-06-09 08-55-54](https://github.com/Dachrono/ToDoList/assets/65233860/491ef95a-bf78-4aac-af5e-ef8a5f968ef0)